<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TimeDlg</class>
 <widget class="QDialog" name="TimeDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>439</width>
    <height>96</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>时间管理</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>日期转时间戳：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDateTimeEdit" name="dateTimeTick">
       <property name="minimumSize">
        <size>
         <width>160</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>160</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="displayFormat">
        <string>yyyy-MM-dd HH:mm:ss</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="tbTimeToTick">
       <property name="text">
        <string>转换</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEditTick"/>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>时间戳转日期：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEditDateTick">
       <property name="minimumSize">
        <size>
         <width>160</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>160</width>
         <height>16777215</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="tbTickToTime">
       <property name="text">
        <string>转换</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEditDate"/>
     </item>
    </layout>
   </item>
   <item row="2" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>日期计算年龄：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDateTimeEdit" name="dateTimeBirth">
       <property name="minimumSize">
        <size>
         <width>160</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>160</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="displayFormat">
        <string>yyyy-MM-dd HH:mm:ss</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="tbCaclOldBirth">
       <property name="text">
        <string>计算</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEditBirth"/>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="resourse.qrc"/>
 </resources>
 <connections/>
</ui>
